### PR TITLE
Instrumentation page

### PR DESCRIPTION
**Pull request by DIANAAJAMBO**
Moved an image from the libraries page to the instrumentation page.

### CODE REVIEW

File: static/img/libraries-instrumentation.svg
Comment (svrnm) [inline static/img/libraries-instrumentation.svg:1]: Move this file to the folder `content/en/docs/concepts/instrumentation`

File: content/en/docs/concepts/instrumentation/libraries.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ and document hooks. Other advantages provided by native instrumentation include:
   various consumption scenarios using a wide variety of well-documented
   OpenTelemetry extensibility points.
 
+
+
```
Comment (svrnm) [inline content/en/docs/concepts/instrumentation/libraries.md:26]:
```suggestion
```

undo this change

File: content/en/docs/concepts/instrumentation/_index.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ aliases: [instrumenting]
 weight: 15
 ---
 
+![ Native Instrumentation vs instrumentation libraries](/img/libraries-instrumentation.svg)
```
Comment (svrnm) [inline content/en/docs/concepts/instrumentation/_index.md:8]: update this with the right URL after you have moved the file to the right location

File: content/en/docs/concepts/instrumentation/libraries.md (hunk 1 above)
Comment (DIANAAJAMBO) [reply content/en/docs/concepts/instrumentation/libraries.md:26]: Removing both lines takes it back to the original (looks better) so let me go with that. Thank you

**Review by svrnm — APPROVED** [review]

File: content/en/docs/concepts/instrumentation/_index.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ aliases: [instrumenting]
 weight: 15
 ---
 
+![ Native Instrumentation vs instrumentation libraries](/libraries-instrumentation.svg)
```
Comment (chalin) [inline content/en/docs/concepts/instrumentation/_index.md:8]:
```suggestion
![ Native Instrumentation vs instrumentation libraries](libraries-instrumentation.svg)
```

File: content/en/docs/concepts/instrumentation/_index.md
Comment (chalin) [inline content/en/docs/concepts/instrumentation/_index.md:1]: The original issue:

- #5519 

said to _move_ the image. So you also need to delete it from https://github.com/open-telemetry/opentelemetry.io/blob/main/static/img/libraries-instrumentation.svg.

File: content/en/docs/concepts/instrumentation/libraries-instrumentation.svg
Comment (chalin) [inline content/en/docs/concepts/instrumentation/libraries-instrumentation.svg:1]: Please trim out the excessive whitespace around the image content.

Comment (chalin) [reply content/en/docs/concepts/instrumentation/libraries-instrumentation.svg:1]: Also, consider renaming this file `native-vs-libraries.svg`.

**Review by chalin — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/instrumentation/libraries-instrumentation.svg
Comment (DIANAAJAMBO) [reply content/en/docs/concepts/instrumentation/libraries-instrumentation.svg:1]: I trimmed the excessive whitespace around the image content and renamed the file to native-vs-libraries.svg. Please let me know if you need any further adjustments.

File: content/en/docs/concepts/instrumentation/_index.md
Comment (svrnm) [reply content/en/docs/concepts/instrumentation/_index.md:1]: @DIANAAJAMBO please take a look

**Comment by svrnm** [comment]
@DIANAAJAMBO please take a look at this comment: https://github.com/open-telemetry/opentelemetry.io/pull/5523#discussion_r1835117995

**Comment by DIANAAJAMBO** [comment]
> @DIANAAJAMBO please take a look at this comment: [#5523 (comment)](https://github.com/open-telemetry/opentelemetry.io/pull/5523#discussion_r1835117995)

Hello @svrnm , I made this change in my last commit, as seen here: https://github.com/open-telemetry/opentelemetry.io/pull/5523/files. However, the original image referenced was merged into the main repository, and I currently lack the necessary rights to update it.
Could you please review my last commit and confirm if everything looks good? If so, we can proceed with the merge.
Thank you.

**Comment by svrnm** [comment]
> > @DIANAAJAMBO please take a look at this comment: [#5523 (comment)](https://github.com/open-telemetry/opentelemetry.io/pull/5523#discussion_r1835117995)
> 
> Hello @svrnm , I made this change in my last commit, as seen here: [#5523 (files)](https://github.com/open-telemetry/opentelemetry.io/pull/5523/files). However, the original image referenced was merged into the main repository, and I currently lack the necessary rights to update it. Could you please review my last commit and confirm if everything looks good? If so, we can proceed with the merge. Thank you.

You copied the file but you didn't move the file: https://github.com/DIANAAJAMBO/opentelemetry.io/blob/librariesPage/static/img/libraries-instrumentation.svg is still existing, so what you need to do is to remove that file and then commit that as a change, git(hub) will then detect that this is a move.

**Comment by DIANAAJAMBO** [comment]
> > > @DIANAAJAMBO please take a look at this comment: [#5523 (comment)](https://github.com/open-telemetry/opentelemetry.io/pull/5523#discussion_r1835117995)
> > 
> > 
> > Hello @svrnm , I made this change in my last commit, as seen here: [#5523 (files)](https://github.com/open-telemetry/opentelemetry.io/pull/5523/files). However, the original image referenced was merged into the main repository, and I currently lack the necessary rights to update it. Could you please review my last commit and confirm if everything looks good? If so, we can proceed with the merge. Thank you.
> 
> You copied the file but you didn't move the file: https://github.com/DIANAAJAMBO/opentelemetry.io/blob/librariesPage/static/img/libraries-instrumentation.svg is still existing, so what you need to do is to remove that file and then commit that as a change, git(hub) will then detect that this is a move.

Okay thank you. I will implement this today.

**Comment by DIANAAJAMBO** [comment]
> > > > @DIANAAJAMBO please take a look at this comment: [#5523 (comment)](https://github.com/open-telemetry/opentelemetry.io/pull/5523#discussion_r1835117995)
> > > 
> > > 
> > > Hello @svrnm , I made this change in my last commit, as seen here: [#5523 (files)](https://github.com/open-telemetry/opentelemetry.io/pull/5523/files). However, the original image referenced was merged into the main repository, and I currently lack the necessary rights to update it. Could you please review my last commit and confirm if everything looks good? If so, we can proceed with the merge. Thank you.
> > 
> > 
> > You copied the file but you didn't move the file: https://github.com/DIANAAJAMBO/opentelemetry.io/blob/librariesPage/static/img/libraries-instrumentation.svg is still existing, so what you need to do is to remove that file and then commit that as a change, git(hub) will then detect that this is a move.
> 
> Okay thank you. I will implement this today.

@svrnm All should be okay now. Help me review. Thanks

File: content/en/docs/concepts/instrumentation/_index.md
Comment (svrnm) [inline content/en/docs/concepts/instrumentation/_index.md:1]: shouldn't this be `content/en/docs/concepts/instrumentation/libraries.md` ?

File: content/en/docs/concepts/instrumentation/native-vs-libraries.svg
Comment (svrnm) [inline content/en/docs/concepts/instrumentation/native-vs-libraries.svg:1]: Can you make sure that this image also work in darmkode, e.g. remove the white background and choose a different color for the "vs" so it is visible on light and dark backgrounds

**Review by svrnm — CHANGES_REQUESTED** [review]
thanks, 2 more comments

**Review by chalin — CHANGES_REQUESTED** [review]
@svrnm & @DIANAAJAMBO - thinking about this more: how about putting this information in a table rather than an SVG? It would be easier to consult, visualize, and render (e.g., dark-mode would be handled automatically; no need to putts around with the SVG colors). WDYT?

**Comment by DIANAAJAMBO** [comment]
> @svrnm & @DIANAAJAMBO - thinking about this more: how about putting this information in a table rather than an SVG? It would be easier to consult, visualize, and render (e.g., dark-mode would be handled automatically; no need to putts around with the SVG colors). WDYT?

Thank you for the suggestion, @chalin . I have already implemented the dark mode change along with the image update, but let’s hear @svrnm's thoughts before finalizing. This way, I can incorporate any necessary adjustments.

File: content/en/docs/concepts/instrumentation/native-vs-libraries.svg
Comment (DIANAAJAMBO) [reply content/en/docs/concepts/instrumentation/native-vs-libraries.svg:1]: Screenshot 2024-12-16 103515

@svrnm Kindly review the image in darkmode.

**Comment by svrnm** [comment]
> > @svrnm & @DIANAAJAMBO - thinking about this more: how about putting this information in a table rather than an SVG? It would be easier to consult, visualize, and render (e.g., dark-mode would be handled automatically; no need to putts around with the SVG colors). WDYT?
> 
> Thank you for the suggestion, @chalin . I have already implemented the dark mode change along with the image update, but let’s hear @svrnm's thoughts before finalizing. This way, I can incorporate any necessary adjustments.

Proposal: we merge this PR with the image for now, since it is there already and it's an incremental update and then we can talk about turning it into a table instead. Another advantage of that is accessibility, since the SVG might not be readable easily as it is right now.

File: content/en/docs/concepts/instrumentation/_index.md
Comment (svrnm) [reply content/en/docs/concepts/instrumentation/_index.md:1]: @DIANAAJAMBO please take a look at this issue as well

Comment (DIANAAJAMBO) [reply content/en/docs/concepts/instrumentation/_index.md:1]: @svrnm  Yes i have implemented this too locally, and only waiting for confirmation to commit the changes. However, my initial confusion came from this change. #5519 . So i thought i was supposed to move it from the libraries page to the instrumentation page as opposed to the instrumentation folder.

Comment (svrnm) [reply content/en/docs/concepts/instrumentation/_index.md:1]: you can commit the changes independent of any further discussions :-)

**Review by svrnm — APPROVED** [review]

**Review by chalin — APPROVED** [review]
- @DIANAAJAMBO - thank you for your work.
- @svrnm - I support your suggestion.
- @DIANAAJAMBO, please create an issue for the followup work, and add a reference to it here.
